app/controllers/admin/organisations/online_booking/motifs_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Admin::Organisations::OnlineBooking::MotifsController < AgentAuthController
+  before_action :set_organisation
+  before_action :set_motif
+
+  def show; end
+
+  def edit; end
+
+  def update
+    @motif.assign_attributes(params.require(:motif).permit(:min_public_booking_delay, :max_public_booking_delay, :rdvs_editable_by_user))
+
+    authorize(@motif, policy_class: Agent::MotifPolicy)
```

Comment:
éventuellement un commentaire pour rappeler que le deuxième authorize est nécessaire dans les updates?
```suggestion
    authorize(@motif, policy_class: Agent::MotifPolicy) # re-authorizing after assigning attributes is important
```

Reply:
j'ai ajouté un commentaire pour préciser justement que c'est pas important ici, et expliquer pourquoi on le met quand même. Ça aurait vraiment pas été sympa pour nous dans le future de dire que c'est important mais pas pourquoi."